docker/cdcstressor/README.md

```diff
@@ -0,0 +1,6 @@
+```
+export CDC_STRESSER_DOCKER_IMAGE=scylladb/hydra-loaders:cdc-stresser-v1
```

Comment:
won't a date be more usable, other wise you need to explain that this version need to be changed, otherwise people would overwrite it by mistake.

Reply:
switched to date.